src/Core/I18N/index.js

```diff
@@ -0,0 +1,47 @@
+const I18N_KEYS = {
+    APP_NAME: "APP_NAME",
+    SEARCH_TITLE: "SEARCH_TITLE",
+    SEARCH_PLACEHOLDER: "SEARCH_PLACEHOLDER",
+    SEARCH_BUTTON: "SEARCH_BUTTON",
+    SORT_BY: "SORT_BY",
+    ALL: "ALL",
+    RELEASE_DATE: "RELEASE_DATE",
+    MOVIE_ID: "MOVIE_ID",
+    MOVIE_RELEASE_DATE: "MOVIE_RELEASE_DATE",
+    MOVIE_TITLE: "MOVIE_TITLE",
+    MOVIE_URL: "MOVIE_URL",
+    MOVIE_WEBSITE: "MOVIE_WEBSITE",
+    MOVIE_OVERVIEW: "MOVIE_OVERVIEW",
+    MOVIE_RUNTIME: "MOVIE_RUNTIME",
+    ADD_MOVIE: "ADD_MOVIE",
+    EDIT_MOVIE: "EDIT_MOVIE",
+    NO_MOVIE_FOUND: "NO_MOVIE_FOUND",
+    DELETE_MOVIE: "DELETE_MOVIE",
+    DELETE_MOVIE_CONFIRM: "DELETE_MOVIE_CONFIRM",
+    TITLE: "TITLE",
+    GENRE: "GENRE",
+    RATING: "RATING",
+    OVERVIEW: "OVERVIEW",
+    RUNTIME: "RUNTIME",
+    RESET: "RESET",
+    SUBMIT: "SUBMIT",
+    CONFIRM: "CONFIRM",
+    EDIT: "EDIT",
+    DELETE: "DELETE", 
+    ERROR_TEXT: "ERROR_TEXT",
+    PAGE_NOT_FOUND: "PAGE_NOT_FOUND",
+    FOUR_OH_FOUR: "FOUR_OH_FOUR",
+    GO_HOME: "GO_HOME"
+}
+
+//Translation Files
+const { EN } = require("./en.js");
```

Comment:
Consider lifting up all imports to the top to follow:
Top - input
Middle - logic
Bottom - output

This way all your files will look the same which greatly improves readability.

Reply:
Thanks Dmitry,

Amendments have been made to the API folder with async methods removed (Although, as you suggested, ESLint has offered a warning that it may be changed back to async!) and .catch clauses added which for now log the error. 

I have tried to lift this import up to the top-level and change to using ES6 Import/Export, but I seem to get errors when I do so ('Cannot read property "APP_NAME" of undefined'). Likely the cause is the method I've used for the index.js file to read the en.js file. Is there any way around this?

Thanks

Reply:
I see.
It's hard to guess at this point what might cause a problem with the import, so the best chance in solving the issue would be to finish with the overall code improvements based on "best practices" and then later see how everything works all together.

Reply:
Regarding the unresolved dependencies, I see that 2 packages "react-i18next" and "i18next" are not added to package.json, so just by checking out the branch I can't get it working without manually installing these packages.